shadan-abdulbasit/style.css

```diff
@@ -0,0 +1,139 @@
+body{
+  margin:0px;
+  padding:0px;
+  font-family:"Helvetica";
+}
+.container{
+  display: flex;
+ 
+} 
+
+aside{
+  
+  flex: 2;
+  background: #3d4f5d;
+  display:flex;
+  align-items:flex-start;
+  text-align: right;
+  color: white;
+  padding-left: 5px;
+  margin-right: 10px;
+  padding: 200px 0;
+  
+}
+
+aside div {
+  padding-left: 30px;
+  position:fixed; 
+
+  
+}
+
+aside div p a{
```

Comment:
try to use classes instead of tag selectors

Reply:
why? isn't the some?